about/index.md

```diff
@@ -5,16 +5,16 @@ weight: 10
 
 <img src="http://i.imgur.com/cADD79P.png" alt="Buoy: Tell your friends where you are and what you need." align="right" width="300" />
 
-Buoy is a free and open source, decentralized tool designed to quickly connect users with trusted friends, family members, advocates, and other allies in times of need.
+Buoy is a crisis response platform that connects families, friends, and teams during emergency situations. 
 
-Buoy is a community-based crisis response system for any emergency in which someone needs help but might not want to -- or can't -- call 911.
+Buoy is a free tool designed to quickly to quickly alert and communicate with a prescribed group of people when assistance is needed.
```

Comment:
Addressed in https://github.com/betterangels/buoy/pull/188/commits/7271f8c6ceb7ac7572de3d4db5220faaa832b842